content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst

```diff
@@ -1,116 +1,144 @@
+checkbox to install it. Multiple shipping connectors may be selected. Then, click :guilabel:`Save`.
+
+.. image:: ../setup_configuration/third_party_shipper/shipping-connectors.png
    :align: center
+   :alt: Options of available shipping connectors in Odoo.
 
-Then click on **Apply**.
+Delivery method
+---------------
 
-Configure the delivery method
------------------------------
+Next, add the carrier details by going to :menuselection:`Inventory app --> Configuration -->
+Shipping Methods`.
 
-To configure your delivery methods, go to the **Inventory** module, click on
-:menuselection:`Configuration --> Delivery Methods`.
+On the :guilabel:`Shipping Methods` page, choose the desired delivery method. For details on
+configuring the third party shipping method, go to:
 
-The delivery methods for the chosen shippers have been automatically
-created.
+.. seealso::
+   - :doc:`Configure third-party carrier <../setup_configuration/third_party_shipper>`
+   - :doc:`DHL credentials <../setup_configuration/dhl_credentials>`
+   - :doc:`Sendcloud credentials <../setup_configuration/sendcloud_shipping>`
+   - :doc:`UPS credentials <../setup_configuration/ups_credentials>`
+
+Then, select the :guilabel:`Get Rate and Create Shipment` option in the :guilabel:`Integration
+Level` field to ensure the shipping label gets printed.
 
-.. image:: labels/labels01.png
+.. image:: labels/integration-level.png
    :align: center
+   :alt: Set the "Get Rate and Create Shipment" option.
+
+Production environment
+~~~~~~~~~~~~~~~~~~~~~~
 
-In the **Pricing** tab, the name of the provider means that the delivery
-will be handled and computed by the shipper system.
+With the delivery method details configured, click the :guilabel:`Test Environment` smart button to
+set it to production environment.
 
-The configuration of the shipper is split into two columns :
+.. warning::
+   Setting the delivery method to :guilabel:`Production` creates **real** shipping labels, and users
+   are at risk for being charged through their carrier account (e.g., UPS, FedEx), **before** users
+   charge customers for shipping. Verify all configurations are correct before launching the
+   delivery method to :guilabel:`Production`.
 
--   The first one is linked to **your account** (develop key, password,...).
-    For more information, please refer to the provider website.
+.. image:: ../setup_configuration/third_party_shipper/production.png
+   :align: center
+   :alt: Show the "Test Environment" smart button.
 
--   The second column varies according to the **provider**. You can choose
-    the packaging type, the service type, the weight unit...
+Print tracking labels
+=====================
 
--  Uncheck **Test Mode** when you are done with the testings.
+Tracking labels are generated after the delivery order (DO) is validated. Begin by creating a
+quotation, adding the shipping cost, confirming the sales order, and then validating the |DO|.
```

Comment:
I genuinely appreciate the concise nature of this instruction, however, do you think a quick mention of where they can do this would be helpful? Like, "Open the :menuselection:`Sales app`, and proceed to..."

Or something like that? What do you think?

Reply:
I'm not sure if specifying how to make a quotation is necessary in this opening paragraph, as the "Add shipping on quotation" section goes into detail on how to do that (: